### ISSUE TITLE
Remove WebsiteContent.content_filepath field

### ISSUE THREAD

**Issue by gsidebo** [mitodl/ocw-studio#268]
Depends on #261. This is included as a separate issue to make for more manageable reviews

**Acceptance criteria:**
- `WebsiteContent.content_filepath` field removed
- Github sync and OCW course site logic updated to drop all references to the removed field
